Examples/Detectors/MagneticField/src/MagneticFieldOptions.cpp

```diff
@@ -66,6 +68,27 @@ void ActsExamples::Options::addMagneticFieldOptions(Description& desc) {
       "option only needs to be set if the field value unit in the field map "
       "file is not `Tesla`. The value must scale from the stored unit to the "
       "equvalent value in `Tesla`.");
+  opt("bf-solenoid-mag-tesla", value<double>()->default_value(0.),
```

Comment:
is 0T the desired default?

Reply:
Not desirable, but in default, the magnetic field is zero. And we only created the solenoid field when this value is not zero.